libraries/stdlib/genglsl/mx_surface_unlit.glsl

```diff
@@ -0,0 +1,8 @@
+
+void mx_surface_unlit(float emission, vec3 emission_color, float transmission, vec3 transmission_color, float opacity, out surfaceshader result)
```

Comment:
What would you think about combining these terms as follows?

```
    result.color = emission * emission_color * opacity;
    result.transparency = mix(vec3(1.0), transmission * transmission_color, opacity);
```

To my eye, that seems more straightforward for the reader to understand, with one assignment each to `result.color` and `result.transparency`.